index.js

```diff
@@ -229,6 +229,15 @@ function emitKey(stream, s) {
     key.name = String.fromCharCode(s.charCodeAt(0) + 'a'.charCodeAt(0) - 1);
     key.ctrl = true;
 
+  } else if (s.length === 1 && s >= '0' && s <= '9') {
+    // number
+    key.name = s;
+
+  } else if (s.length === 1 && '~!@#$%^&*()_+'.indexOf(s) !== -1) {
```

Comment:
good point. would you consider merging, to resolve this for US-ASCII keyboards, then we can work on support for non-US-ASCII? No worries either way, I'm happy to try to figure out a more robust solution.